.github/codeql-analysis.yml

```diff
@@ -0,0 +1,71 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
```

Comment:
I wonder what will happen once we have other languages. Will this still work?

Reply:
we just need to update line 35. This detection is not done all the time but only in the beginning. Maybe we should remove the comments until line 12?

Reply:
Correct. Hence I think the comments are confusing. I would rather point to line 35.
Anyhow, I still wonder what happens if we add other code like "go" or so, since `.github/codeql-analysis.yml` is on root level. Anyhow, we will se...

Reply:
I _guess_ it does analysis based on the language so we _should_ be fine. I have removed the comments